embedded-tests/src/test/java/org/apache/druid/testing/embedded/msq/EmbeddedMSQRealtimeQueryTest.java

```diff
@@ -141,12 +163,38 @@ public EmbeddedDruidCluster createCluster()
         .addResource(kafka)
         .addServer(coordinator)
         .addServer(overlord)
-        .addServer(indexer)
-        .addServer(broker)
-        .addServer(historical)
         .addServer(router);
   }
 
+  @BeforeAll
+  @Override
+  protected void setup() throws Exception
+  {
+    cluster = createCluster();
+    cluster.start();
```

Comment:
This appears to be copied from the superclass; you can replace it with `super.setup()`